scripts/twister

```diff
@@ -684,6 +684,13 @@ structure in the main Zephyr tree: boards/<arch>/<board_name>/""")
         help="Execute time-consuming test cases that have been marked "
              "as 'slow' in testcase.yaml. Normally these are only built.")
 
+    parser.add_argument(
+        "--short-build-path",
+        action="store_true",
+        help="Create shorter build directory paths based on symbolic links. "
+             "This option can be useful on Windows OS where path length limit "
+             "occurs.")
```

Comment:
I think it should be make it clearer what this implies.

How about adjusting to something like this:
```suggestion
            "The shortened build path will be used by CMake for generating the build system and executing the build."
             "Use this option if you experience build failures related to path length, for example on Windows OS.")
```